pyproject.toml

```diff
@@ -21,10 +21,9 @@ profile = "black"
 [tool.mypy]
 warn_unused_configs = true
 warn_unused_ignores = true
-python_version = "3.6"
+python_version = "3.9"
```

Comment:
Targeting 3.9 gives better typing for `importlib.resources` (instead of `importlib_resources`), which is why I changed this here.

Reply:
Fair enough. I've also seen the odd place where we still use old-ish Python versions in checks. Will try and change as I "navigate around" ...

Reply:
I usually run on the latest Python but target the oldest Python supported for checks. For high profile projects, sometimes we run mypy twice, once with oldest targeted, and once with newest targeted. Normally, oldest is fine, but if there's a reason checking is harder on the oldest (like here), I don't think there's a huge loss - we still have unit tests for older versions, we just won't get mypy immediately telling us if we use something from a newer Python.